docker-nginx/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM alpine:latest
+
+RUN apk update
+RUN apk --no-cache add nginx  php83-mbstring php83-fpm php83-mysqli php82-mysqlnd php83-session curl unzip
```

Comment:
Miksi mysqlnd:n PHP 8.2:n paketti?

Reply:
:) Kato, varmaankin vahinko. Ei ole toivomuksia asioiden suhteen. Monessa opensource ohjelmassa näkee Dockerfile (ja docker-compose.yml) samassa projektissa, jos jollekulle docker tuttu niin saa nopeasti ja vaivattomasti käyttöön. Tosin webhotel käyttäjälle pari ylimääräistä filua (dockerfile, docker-compose, conffi-esimerkki palvelimelle).

En muista laitoinko tuosta ligttpd/nginx, mutta ensiksi tein lighttpd:llä ja jälkeenpäin saman nginx:llä. Lighttpd:n 'vikana' oli se, että siinä ei saa helposti erroreita stdouttiin (docker logiin) vaan pitää käyttää kikkailua / tiedostoa. Nginx saa suoraan stdouttiin. Nginx lienee huomattavasti suositumpi, joten myös käyttäjät osannevat konffata ehkä sen helpommin. Saman voi tehdä myös apachella, mutta itse en ole vuosiin konffannut apachea.

Ei docker tosiaan ole riippuvainen MLinvoicen versioinnista, mutta jos tuo Dockerfile roikkuu projektin mukana niin senhän voi silloin  rakentaa  (docker build) ettei ydintä tarvitse hakea zippinä ulkopuolelta. Ja .dockerignorella voidaan määritellä myös jos ei jotain haluta dockkeriin mukaan...

Reply:
Joo, no ei kai tuossa ongelmaa sinänsä pitää sitä mukana. Mutta voisitko poistaa tuon php 8.2 -paketin vielä?

Reply:
Tuosta voi vaihtaa numeron 2 => 3. Äkkiseltään en muista voiko tuon koko paketin poistaa, nyt en ole dockerin äärellä.

Reply:
Poistin sen tosiaan jo aiemmin, kun ei sitä pitäisi tarvita ainakaan eksplisiittisesti.